packages/tag/index.demo.tsx

```diff
@@ -31,10 +31,10 @@ export class Demo extends Component<void> {
         <legend>{Tag.is}</legend>
 
         <div>
-          <Tag onTagClose={this.tagClose}>Tag one</Tag>
-          <Tag onTagClose={this.tagClose}>Tag two</Tag>
-          <Tag onTagClose={this.tagClose}>Tag three</Tag>
-          <Tag onTagClose={this.tagClose}>Tag four</Tag>
+          <Tag onTagClose={this.tagClose} label="Taone"></Tag>
```

Comment:
don't use closing tags if element has no children. this rule should be done in `tslint` ( currently tslint definition is very poor. I'll update it later )

instead:
`<Tag onTagClose={this.tagClose} label="Taone"></Tag>`

do:
`<Tag onTagClose={this.tagClose} label="Taone" />`